warehouse/query-core/src/main/java/datawave/query/iterator/UniqueIterator.java

```diff
@@ -0,0 +1,54 @@
+package datawave.query.iterator;
+
+import com.google.common.base.Predicate;
+import datawave.query.attributes.Document;
+import datawave.query.attributes.UniqueFields;
+import datawave.query.common.unique.UniqueUtil;
+import datawave.query.iterator.profile.FinalDocumentTrackingIterator;
+import datawave.query.transformer.UniqueTransform;
+import org.apache.accumulo.core.data.Key;
+import org.apache.log4j.Logger;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.Map;
+
+public class UniqueIterator {
```

Comment:
Add a short description, even if it's a simple as "delegates unique operations to UniqueUtil"